app/database/operator/server_data_operator/transformers/channel.ts

```diff
@@ -42,20 +42,24 @@ export const transformChannelRecord = ({action, database, value}: TransformerArg
 
         // for DM & GM's  channels do not override the display name
         // until we get the new info if there is any
-        let displayName;
-        if (raw.type === General.DM_CHANNEL && (record?.displayName || raw.display_name)) {
-            displayName = raw.display_name || record?.displayName;
-        } else if (raw.type === General.GM_CHANNEL) {
-            const rawMembers = raw.display_name.split(',').length;
-            const recordMembers = record?.displayName.split(',').length || rawMembers;
-
-            if (recordMembers < rawMembers && record.displayName) {
-                displayName = record.displayName;
-            } else {
-                displayName = raw.display_name;
+        let displayName = '';
+        switch (raw.type) {
+            case General.DM_CHANNEL:
+                displayName = raw.display_name || record?.displayName;
```

Comment:
Do we want `||` or `??`. Maybe for display name we want `||`, since displayName is not supposed to be set as the empty string, but I wanted to make sure about that.

Reply:
for some reason, there seems to be that this is still being set as empty when the WS reconnects, will have to look further into this